samples/snippets/mf_explicit_model_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (t
+# you may not use this file except in compliance wi
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in
+# distributed under the License is distributed on a
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, eit
+# See the License for the specific language governi
+# limitations under the License.
+
+
+def test_explicit_matrix_factorization(random_model_id: str) -> None:
+    your_model_id = random_model_id
+    # [START bigframes_dataframes_bqml_mf_explicit_create]
+    from bigframes.ml import decomposition
+    import bigframes.pandas as bpd
+
+    # Load data from BigQuery
+    bq_df = bpd.read_gbq(
+        "bqml_tutorial.ratings", columns=("user_id", "item_id", "rating")
```

Comment:
Unfortunately, the samples tests are failing with this error:

```
>           raise exceptions.from_http_response(response)
E           google.api_core.exceptions.NotFound: 404 GET https://bigquery.googleapis.com/bigquery/v2/projects/python-docs-samples-tests/datasets/bqml_tutorial/tables/ratings?prettyPrint=false: Not found: Table python-docs-samples-tests:bqml_tutorial.ratings
```

This is because the samples tests run in several different GCP projects beside the `bigframes-*` ones we use for our other testing. We have a few options. I would recommend putting this data somewhere easier accessible by following the instructions at go/cloud-sample-data I'll actually give this a try. Maybe we can avoid the "upload" step altogether.

Reply:
Actually, nevermind:

>      * The user may not redistribute the data without separate permission.

I think instead we'll need a "upload the movielens data" snippet to avoid redistributing these data.

Reply:
I have added two more snippets in 7f2d7f672c8ec085dcd384ee4eb03ce88e5ba98a

These create the dataset and the tables from the zipfile just like the [commands in the docs](https://cloud.google.com/bigquery/docs/bigqueryml-mf-explicit-tutorial#upload_the_movielens_data) do.